resources/build/build-utils-ci.inc.sh

```diff
@@ -141,9 +143,68 @@ function _builder_write_npm_version() {
           . +
           (try { dependencies: (.dependencies | to_entries | . + map(select(.key | match("@keymanapp/.*")) .value |= $VERSION_WITH_TAG) | from_entries) } catch {}) +
           (try { devDependencies: (.devDependencies | to_entries | . + map(select(.key | match("@keymanapp/.*")) .value |= $VERSION_WITH_TAG) | from_entries) } catch {}) +
-          (try { bundleDependencies: (.bundleDependencies | to_entries | . + map(select(.key | match("@keymanapp/.*")) .value |= $VERSION_WITH_TAG) | from_entries) } catch {}) +
           (try { optionalDependencies: (.optionalDependencies | to_entries | . + map(select(.key | match("@keymanapp/.*")) .value |= $VERSION_WITH_TAG) | from_entries) } catch {})
         ' > "${line}_"
       mv -f "${line}_" "$line"
     done
+}
+
+#
+# Due to https://github.com/npm/cli/issues/3466, we manually create all
+# bundleDependencies (__NOT__ bundledDependencies, beware typos) from
+# the target's package.json in its node_modules folder. Must run from
+# the target's folder.
+#
+function _builder_prepublish() {
+  mkdir -p node_modules/@keymanapp
+  local packages=($(cat package.json | "$JQ" --raw-output '.bundleDependencies | join(" ")'))
+  local package
+
+  # For each @keymanapp/ package, we'll do a local symlink, note that Windows
+  # mklink is internal to cmd!
+  for package in "${packages[@]}"; do
+    if [[ $package =~ ^@keymanapp/ ]]; then
+      # Creating local symlink under node_modules
+      local link_source=node_modules/$package
+
+      # lookup the link_target from top-level package.json/dependencies
+      local link_target="$(cat "$KEYMAN_ROOT/builder_package_publish.json" | jq -r .dependencies.\"$package\")"
```

Comment:
Do we want the platform agnostic call?
```suggestion
      local link_target="$(cat "$KEYMAN_ROOT/builder_package_publish.json" | "$JQ" -r .dependencies.\"$package\")"
```